objects/date.js

```diff
@@ -18,6 +18,11 @@ module.exports = class Date extends global.Date {
 		"December"
 	];
 
+	/**
+	 * Returns the suffix ("st", "nd", "rd", and "th") for a given day.
+	 * @param {number} number Day of the month (1..31)
+	 * @returns {string}
```

Comment:
to be more precise, the return type should either be a JSDoc enum or its own `@typedef` as the return value is really just one of these four string values.